app/src/main.py

```
@@ -504,8 +765,13 @@ def run(self):
     except InvalidConfigFile as e:
         app_logger.fatal(e.message)
         sys.exit(1)
+    except ValueError as e:
+        app_logger.fatal(f'Configuration error: {e}')
+        sys.exit(1)
     except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)